README.md

```diff
@@ -1,66 +1,144 @@
 # Ember JSON API Docs [![Build Status](https://travis-ci.org/ember-learn/ember-jsonapi-docs.svg?branch=master)](https://travis-ci.org/ember-learn/ember-jsonapi-docs)
 
-If you are looking for the app behind https://emberjs.com/api/, visit
-[ember-api-docs](https://github.com/ember-learn/ember-api-docs) instead. This ember-jsonapi-docs
-repository is internal tooling that is not required to run the ember-api-docs app locally.
+This is an internal tool for generating API docs for the Ember.js framework and exposing it through [JSON:API](http://jsonapi.org/) for various applications (e.g. https://api.emberjs.com/). The tool allows you to:
 
-`ember-jsonapi-docs` is for turning code comments in [ember.js](https://github.com/emberjs/ember.js) into
-[json api](http://jsonapi.org/) compliant data for use in various applications seeking to use the Ember API.
+- Generate API docs from [YUIDoc](http://yui.github.io/yuidoc/syntax/index.html) comments in the [emberjs/ember.js](https://github.com/emberjs/ember.js/) and [emberjs/data](https://github.com/emberjs/data) repositories in [YUIDoc](http://yui.github.io/yuidoc/) and [JSON:API](http://jsonapi.org/) formats.
+- Publish docs to an Amazon S3 bucket (`s3://api-docs.emberjs.com`).
+- Expose API docs in the JSON:API format through API.
 
-The script pulls yuidoc build output from all Ember versions from Amazon S3, converts it to json api format and creates an archive. It can also be run to build jsonapi docs from a local copy of ember.js.
+All the generated files are stored in the `tmp` folder under the project root:
 
-## Running the app
+```
+tmp
+├── json-docs   // JSON:API-comlaint docs generated locally from YUIDoc files in `s3-docs`.
+├── rev-index   // Rev index files used for generating JSON:API-comlaint docs.
+├── s3-docs     // YUIDoc docs generated locally or downloaded from s3://api-docs.emberjs.com.
+```
 
-1. Fork/Clone [ember-jsonapi-docs](https://github.com/ember-learn/ember-jsonapi-docs)
-1. Run `yarn`
-1. Install the [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
-1. Set up AWS access
+> ℹ️ **NOTE:** If you are looking for the app behind https://api.emberjs.com/, visit [ember-api-docs](https://github.com/ember-learn/ember-api-docs) instead.
 
-   ```shell
-   export AWS_ACCESS_KEY_ID=xxxxxx
-   export AWS_SECRET_ACCESS_KEY=xxxxx
-   ```
+## Prerequisites
 
-   The app accesses builds.emberjs.com (an Amazon S3 bucket) in read-only mode, which is public. This requires any valid AWS credentials.
```

Comment:
We should probably keep this somewhere.

Reply:
Thank you for the careful review, @Gaurav0! It seems that this information is not relevant anymore: the tools currently uses NPM instead of `builds.emberjs.com` now.

Here is where this was introduced: https://github.com/ember-learn/ember-jsonapi-docs/commit/68f10fdf5ed23fc2acf019751795ea93fee8cdce#diff-9e7e72927fba2b056ef40fa6fa01bf1aL22